src/Doctrine/ORM/PixelEventRepository.php

```diff
@@ -79,6 +94,21 @@ protected static function applyDelay(QueryBuilder $qb, int $delay): void
         }
     }
 
+    public function resetFailedByPixel(PixelInterface $pixel): void
+    {
+        $this->createQueryBuilder('o')
+            ->update()
+            ->set('o.state', ':initialState')
```

Comment:
Should the bulk identifier also be reset? Or other properties?

Reply:
> Should the bulk identifier also be reset? 
No, it will be set to the other specific value when `assignBulkIdentifierToPendingConsented` will be called.

> Or other properties?
No